ui/app/AppLayouts/Wallet/views/TokenSelectorAssetDelegate.qml

```diff
@@ -0,0 +1,133 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+ItemDelegate {
+    id: root
+
+    required property string tokensKey
+    required property string name
+    required property string symbol
+    required property double currentCurrencyBalance
+    // expected structure: balancesModel -> model.balances submodel [chainId: int, balance: BigIntString] + flatNetworks [account:string, iconUrl: string]
+    required property var balancesModel
+
+    property var formatCurrencyAmount: (balance) => "IMPLEMENT ME"
```

Comment:
do you want to print this or?

Reply:
No, that's a hint for the developer/user that this method is really meant to be overridden (because we can't make it `required`)

Reply:
And I removed it now :)